src/screens/MainApp/Home/Dashboard/CreateDiagnoseBoard/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { View, Text, TouchableOpacity } from 'react-native'
+import styles from './styles'
+
+const CreateDiagnoseBoard = ({ handleGoToCreateDiagnoseScreen }) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.title}>
+        <Text style={styles.text}>
```

Comment:
I created a reusable component in helpers "AppText" which has a consistent style (roboto 14). It also has the property of being customizable, so you could only add the font size 18 adhoc.

It's something minor but you could save a little verbosity from applying styles.